controllers/admin/AdminProductsController.php

```diff
@@ -30,8 +30,9 @@
  */
 class AdminProductsControllerCore extends AdminController
 {
-    /** @var int Max image size for upload
-     * As of 1.5 it is recommended to not set a limit to max image size
+    /**
+     * @var int Max image size for upload
+     *          As of 1.5 it is recommended to not set a limit to max image size
```

Comment:
indentation problem or github fail ?

Reply:
This is how a comment should be written when it concerns the line above, otherwise, it's a new line.